src/nimdowpkg/windowmanager.nim

```diff
@@ -343,9 +343,21 @@ proc mapConfigActions*(this: WindowManager) =
     this.focusNextMonitor()
 
   createControl(keycode, "goToTag"):
-    let tag = this.selectedMonitor.keycodeToTag(keycode)
+    var tag = this.selectedMonitor.keycodeToTag(keycode)
+    if this.selectedMonitor.previousTag != -1 and this.selectedMonitor.selectedTag.id == tag.id:
+      let currentTag = tag.id
+      tag = this.selectedMonitor.keycodeToTag(this.selectedMonitor.previousTag+10)
```

Comment:
We can avoid doing this if we use an actual tag instead of an int - Tags are ref objects so it's fine

Reply:
I tried just modifying id key for tag object, without +10, but it did not work.

Reply:
I mean using the type Tag instead of an int. You wouldn't need to modify the id, that shouldn't change

Reply:
Oh, good point.  I will rewrite it tomorrow :+1: